Even though the doc suggest that an s3 ext_pillar will get credentials from the EC2 meta-data, this appears not to be working in 2015.8.3.

It is possible this may be a handoff issue with another part of the code as I see the following error in the logs (that is not really reported as an ERROR)

```
2015-12-21 22:44:37,691 [urllib3.connectionpool                              ][DEBUG   ][20158] "GET /?prefix=pillar HTTP/1.1" 403 None
```

@stanislavb may know something about this?
